tests/components/azure_devops/test_sensor.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for init of Azure DevOps."""
+
+import pytest
+
+from homeassistant.components.azure_devops.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.usefixtures(
+    "entity_registry_enabled_by_default",
+    "mock_devops_client",
+)
+async def test_sensors(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
```

Comment:
```suggestion
@pytest.mark.usefixtures("entity_registry_enabled_by_default")
async def test_sensors(
    hass: HomeAssistant,
    mock_config_entry: MockConfigEntry,
    entity_registry: er.EntityRegistry,
    mock_devops_client: AsyncMock,
) -> None:
```

I acuatally have no clue why I always add `entity_registry_enabled_by_default` as annotation and the rest as parameter, but 🤷🏻